src/shared/options/readOptions.ts

```diff
@@ -238,3 +243,37 @@ export async function readOptions(
 		options: augmentedOptions,
 	};
 }
+
+async function getBase(): Promise<OptionsBase> {
+	const commonScripts = ["lint:knip", "should-semantic-release", "test"];
```

Comment:
[Style] Nit: A `Set` would be more canonical here. I've been meaning to look at enabling https://github.com/sindresorhus/eslint-plugin-unicorn/blob/2f77a236777c89314d91988d350424127696fec7/docs/rules/prefer-set-has.md.